client/components/cocktail-list.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Cocktails extends React.Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      list: props.cocktails
+    };
+  }
+
+  _addCocktails () {
+    return this.props.cocktails.map(function (cocktail) {
+      return <li>{cocktail.name}</li>;
```

Comment:
Since we have IDs for each cocktail, why don't we render them as links? (Mostly just to make it easier to remember which IDs have cocktails attached, when developing...)
```
return (
  <li>
    <a href={`cocktails/${cocktail.id}`}>{cocktail.name}</a>
  </li>
);
```